instruction.md

```diff
@@ -0,0 +1,86 @@
+## Overview
+
+Face Recognition system is a way of recognizing a human face through technology. A facial recognition system uses facial features from a photograph and compares the information with a database of known faces to find a match.
+
+So how does facial recognition work?
+
+Step 1. You need to sign up to the system (First user in CompareFace admin has Owner role, but it is possible to change the role) and then LogIn with created account or just use the existing one. After that system redirects you to the main page.
+
+Step 2. Create an application (left section) with "Create" link at the bottom of the page. An application is where you can create and manage your face collections.
+
+Step 3. Enter you application with double click on the name of the application. Here you will have two possibilities. The first one is to add new users to your application and manage permissions (Owner and Administrator roles already have access to any application without invite, user role doesn't.) The second one is to create face collections.
```

Comment:
`Owner and Administrator roles` - let's call it "Global Owner and Administrator roles"